src/gracenotegroup.ts

```diff
@@ -14,19 +14,33 @@ import { Beam } from './beam';
 import { StaveTie } from './stavetie';
 import { TabTie } from './tabtie';
 import { StaveNote } from './stavenote';
+import { Note } from './note';
 
-// To enable logging for this class. Set `Vex.Flow.GraceNoteGroup.DEBUG` to `true`.
-function L(...args) {
+// To enable logging for this class. Set `GraceNoteGroup.DEBUG` to `true`.
+function L(...args: string[]) {
```

Comment:
This should be `any`. It goes right into `console.log`.

Reply:
ok, done